### PR TITLE
Rating

### PR DESCRIPTION
**Pull request by Gautam2305**
completed rating components

Screenshot (95)

### CODE REVIEW

File: stylesheets/CSS/variables.css [hunk 1]
```diff
@@ -8,6 +8,7 @@
     --st-background-grey: #cbd5e1;
     --st-dark-grey: #94a3b8;
     --st-light-blue: #78d3fa;
+    --st-light-pink: #fce7f3;
```
Comment (shubhamkumar648) [inline stylesheets/CSS/variables.css:12]: you can also put value of colur in hsl

File: stylesheets/Components/modal.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.modal-container{
+    max-width: 50%;
+    margin: auto;
+    background-color: var(--st-light-pink);
+    padding: 15px;
+}
+.modal-info, .modal-btn{
+    padding: 20px;
```
Comment (shubhamkumar648) [inline stylesheets/Components/modal.css:8]: use rem unit for better responive

**Review by shubhamkumar648 — COMMENTED** [review]
reaming looks god

File: stylesheets/CSS/variables.css (hunk 1 above)
Comment (Gautam2305) [reply stylesheets/CSS/variables.css:12]: i am more familiar with hex codes as of now...... the hsl values were taken from another component library....i might add hsl while refactoring my code

File: stylesheets/Components/modal.css (hunk 2 above)
Comment (Gautam2305) [reply stylesheets/Components/modal.css:8]: my root font-size is 20px,  and i don't want the padding to change as it was looking a bit odd at first when i used rems......it's less than 1 rem anyway...so i let it be

Comment (Gautam2305) [reply stylesheets/Components/modal.css:8]: thankyou for taking time out to review❤️

**Pull request merged**